src/devices/cpu/es5510/es5510.h

```diff
@@ -12,12 +12,11 @@
 
 #pragma once
 
-// TODO : Not verified
-#define DRAM_SIZE (1<<20)
-#define DRAM_MASK (DRAM_SIZE-1)
-
 class es5510_device : public cpu_device {
 public:
+	// TODO : Not verified
+	static constexpr DRAM_SIZE (1<<20);
+	static constexpr DRAM_MASK (DRAM_SIZE-1);
```

Comment:
You need to test these changes - this requires you to supply a type and use the `=` sign.